tests/std/tests/P0718R2_atomic_smart_ptrs/test.cpp

```diff
@@ -222,6 +557,25 @@ void ensure_nonmember_calls_compile() {
     }
 }
 
+template <class AtomicType>
+void ensure_member_calls_compile() {
+    AtomicType instance;
+    const AtomicType constInstance;
+    auto loaded = instance.load();
+    loaded      = constInstance.load();
+    loaded      = instance;
+    loaded      = constInstance;
+    instance.store(loaded);
+    loaded = instance.exchange(instance);
+    loaded = instance.exchange(constInstance);
+    if (instance.compare_exchange_weak(loaded, constInstance)) {
+        // intentionally empty
+    }
+    if (instance.compare_exchange_strong(loaded, constInstance)) {

```

Comment:
No change requested: We conventionally add newlines between non-chained `if`-statements. However, this was copied from pre-existing code immediately above, and it's not worth resetting testing by pushing a commit.